AdobeStockImageAdminUi/Controller/Adminhtml/Asset/GetDetails.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImageAdminUi\Controller\Adminhtml\Asset;
+
+use Magento\AdobeStockAssetApi\Model\Asset\Command\LoadByIdsInterface;
+use Magento\Backend\App\Action;
+use Magento\Framework\App\Action\HttpPostActionInterface;
+use Magento\Framework\Controller\Result\Json;
+use Magento\Framework\Controller\ResultFactory;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Backend controller for retrieving asset information by adobeId
+ */
+class GetDetails extends Action implements HttpPostActionInterface
```

Comment:
I think it's better to introduce the GetMediaGalleryAsset controller to retrieve media gallery asset by `adobe_id` to avoid double request.